### DISCUSSION TITLE
Security Scoring

### DISCUSSION THREAD

**Discussion by razaina** [OWASP/mastg#2435] [Ideas]
Hi,

wouldn't it be useful to provide a base security score for each MASTG ? 

Something like the CWSS score [https://cwe.mitre.org/cwss/cwss_v1.0.1.html](https://cwe.mitre.org/cwss/cwss_v1.0.1.html), [https://www.cwss-score.info/](https://www.cwss-score.info/)?

It can be customized later on but at least we have a base score.

**Comment by cpholguera** [comment]
Hi @razaina, excellent timing! Stay tuned to our news and social channels this week. Spoiler: We'll be releasing an initial proposal for a mobile app security score including next steps. We'd love your feedback!

**Reply by cpholguera** [reply]
There you go: https://twitter.com/OWASP_MAS/status/1704540140834586792?t=31iGMgAe4-QpPj0nkzAalA&s=19

**Reply by razaina** [reply]
Haha love it, perfect timing 👌
